### PR TITLE
fix(bitget) - ws tests

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei PR ready

File: ts/src/pro/bitget.ts [hunk 1]
```diff
@@ -682,7 +682,8 @@ export default class bitget extends bitgetRest {
             stored = new ArrayCache (limit);
             this.trades[symbol] = stored;
         }
-        const data = this.safeValue (message, 'data', []);
+        let data = this.safeList (message, 'data', []);
+        data = data.reverse (); // fix chronological order
```
Comment (carlosmiei) [inline ts/src/pro/bitget.ts:686]: @ttodua, instead of doing this, why not iterate from the end? seems more efficient

Comment (ttodua) [reply ts/src/pro/bitget.ts]: travis throws unrelated issue

**Comment by ttodua** [comment]
travis passes

**Pull request merged**